axelor-bank-payment/src/main/resources/i18n/messages_fr.csv

```diff
@@ -938,10 +938,12 @@
 "The moves %s couldn't be reversed because these are linked to a bank reconciliation with status validated","Les écritures %s ne peuvent être extournées car elle sont liées à un rapprochement bancaire au statut validé",,
 "The payment status will not update.","Le statut du paiement ne sera pas mis à jour.",,
 "The receiver bank details currency is not compatible with the currency in bank order.","La devise du RIB destinataire n'est pas compatible avec la devise de l'ordre bancaire",,
-"The receiver bank details is inactive.","Le RIB destinataire est inactif.",,
+"The receiver bank details for the line %s is inactive.","Le RIB destinataire pour la ligne %s est inactif.",,
+"The receiver bank details is inactive.",,,

```

Comment:
Please check why is this line still here but without the translation, if it is not used anywhere please remove it